docs/4-Commands/kool-share.md

```diff
@@ -23,5 +23,4 @@ kool share
 
 ### SEE ALSO
 
-* [kool](kool)	 - Cloud native environments made easy
-
+-   [kool](kool) - Cloud native environments made easy
```

Comment:
run `kool run make-docs` to regenerate it as it was before...

Reply:
alright!